pihole

```diff
@@ -300,6 +300,29 @@ Branches:
   checkout "$@"
 }
 
+tricorderFunc() {
+  if [ ! -p "/dev/stdin" ]; then
+    echo "Please do not call Tricorder directly."
+    exit 1
+  fi
+
+  if command -v openssl &> /dev/null; then
```

Comment:
Should we add in a check like `nc -v -z -w5 tricorder.pihole.net 9998` at the start to check and see if the port is open and accesible? Will retcode 0|1 for status...